app/services/import/legacy/transform-contacts.service.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+/**
+ * Transforms NALD company data into a valid object that matches the WRLS structure
+ * @module ImportLegacyTransformContactsService
+ */
+
+const FetchContactsService = require('./fetch-contacts.service.js')
+const ContactPresenter = require('../../../presenters/import/legacy/contact.presenter.js')
+const CompanyContactPresenter = require('../../../presenters/import/legacy/company-contact.presenter.js')
+const ImportContactValidator = require('../../../validators/import/contact.validator.js')
+
+/**
+ * Transforms NALD company data into a validated object that matches the WRLS structure
+ *
+ * @param {string} regionCode - The NALD region code
+ * @param {string} licenceId - The NALD licence ID
+ * @param {object[]} transformedCompanies
+ *
+ * @returns {Promise<object>} an object representing an array of valid WRLS transformed companies and
+ * an array of companies from the db
```

Comment:
My bad. Didn't spot on the `TransformCompaniesService` that there is a documented return when we don't actually have one, and no doubt this is here because of that.

```suggestion
```